quickfixj-core/src/main/java/quickfix/ThreadedSocketInitiator.java

```diff
@@ -82,11 +82,12 @@ public void stop() {
     }
 
     public void stop(boolean forceDisconnect) {
-        stopInitiators();
         logoutAllSessions(forceDisconnect);
+        stopSessionTimer();
```

Comment:
Yeah, call to stopSessionTimer() has been removed in https://github.com/quickfix-j/quickfixj/commit/5f365094aec7b529f91ae518313c8b12517bdede